doctor/urls.py

```diff
@@ -35,4 +35,5 @@
         views.get_document_number,
         name="document-number-pdf",
     ),
+    path("utils/x-ray/pdf/", views.xray, name="xray-pdf"),
```

Comment:
I think it's worth giving this a more human name, since most people won't know what xray is. Maybe:

```suggestion
    path("utils/check-redactions/pdf/", views.xray, name="xray-pdf"),
```